static/partials.css

```diff
@@ -44,13 +44,7 @@ form {
     margin-left: auto;
 }
 
-.navbar m logo {
-    padding-left: 10px;
-    font-family: "Shrikhand";
-    font-size: 50px;
-}
-
-.navbar m motto {
+.navbar div motto {
```

Comment:
I've personally become a big fan of best-effort BEM-ish (Block Element Modifier) CSS selectors. Lots of google material on it - most of it makes it sound more complicated than it is - so i included an example.

It seems initially overly verbose (preprocessors like SCSS/LESS can help, but even w/o I still use), and I was not keen on it when my frontend engineer started to incorporate it,  but once I really started using it myself being explicit up front, keeping the selector nesting level low and never relying on the structure (for example if a div changes to a span) makes it worthwhile and saves a _ton_ if time when trying to understand how the final style was constructed and where the CSS lives (even w/ sourcemaps its nice to tell whats going on from the classnames in the html alone and Find Usages is very easy)

Something like this would look like:

```css
.navbar { /* your Block */
  color: #333 
  /* ...and the rest */ 
}

.navbar__tagline {  /* Block-Elements, delimited by __ - states tagline is an element of block: navbar */
  display: block; /* ... */ 
}

.navbar__linktext { /* another element, this formatting doesn't need multiple __foo__bar nesting */
  margin: 0px 10px; 
  padding: 5px;
}

/* 
ex of a "--modifier" if hover was being added via JS for example
the modifier class is _added_ in addition to its base class (along the lines of btn btn-primary)
in this case it would have the class="navbar__linktext nabar__linktext--hover" when "modified"
*/
.navbar__linktext:hover,  
.navbar__linktext--hover { 
  background-color: #ddd;
}

.framebox { /* starting a new, distict Block, when you do this is situational, but more focused blocks are generally better */ }
```

Reply:
I am a fan of this I didn't make the header so I didn't change it too much. just tried to change to real elements

I also think we should change to SCSS to make things simpler.
We should do that as its own refactor